City-of-Helsinki/drupal-helfi-platform-config
# UHF-2828: remove Matomo modules

Removes Matomo modules and their configuration.

How to test:

- Create fresh platform instance:
  - `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
  - `cd hel-platform`
- Checkout this branch: `composer require drupal/helfi_platform_config:dev-UHF-2828_remove_matomo`
- Run `make new`
- After the install, make sure that there is no "Matomo" or "Matomo reports" modules in the system
